Assets/cqui_settings.sql

```diff
@@ -64,6 +64,7 @@ INSERT OR REPLACE INTO CQUI_Settings -- Don't touch this line!
       ('CQUI_ShowTechCivicRecommendations', 1), -- Shows the advisor recommendation in the techs/civics tree/panel
       ('CQUI_ShowImprovementsRecommendations', 0), -- Shows the advisor recommendation for the builder improvements
       ('CQUI_ShowCityDetailAdvisor', 0), -- Shows the advisor recommendation in the city detail panel
+	  ('CQUI_ShowDiploBanner', 1), -- Shows the Diplomacy Banner
```

Comment:
Bad indentation here also

Reply:
I know, i use Notepad++ and it always creates a mess with CQUI... why Chao decided to go with spacing indentation is beyond me, he was about to transition the mod to tab indentation when the whole Summer debacle happened, and he left... will change these, but we should discuss about how to deal with it in the future...

Reply:
Again: why did you block uploads to the CQUI master? It makes contributing more annoying, and will complicate things for the other contributors... it was fine before. Can you open it up again?